ext/standard/tests/streams/stream_set_chunk_size.phpt

```diff
@@ -24,45 +24,89 @@ class test_wrapper {
     }
 }
 
-var_dump(stream_wrapper_register('test', 'test_wrapper'));
+var_dump(stream_wrapper_register('testlocal', 'test_wrapper'));
+var_dump(stream_wrapper_register('testurl', 'test_wrapper', STREAM_IS_URL));
 
-$f = fopen("test://foo","r");
+foreach (['testlocal', 'testurl'] as $stream_type) {
+    $f = fopen("$stream_type://foo","r");
 
-/* when the chunk size is 1, the read buffer is skipped, but the
- * the writes are made in chunks of size 1 (business as usual)
- * This should probably be revisited */
-echo "should return previous chunk size (8192)\n";
-var_dump(stream_set_chunk_size($f, 1));
-echo "should be read without buffer (\$count == 10000)\n";
-var_dump(strlen(fread($f, 10000)));
-echo "should have no effect on writes\n";
-var_dump(fwrite($f, str_repeat('b', 3)));
+    /* when the chunk size is 1, the read buffer is skipped, but the
```

Comment:
Minor duplication issue: this line ends with `the` and the next line starts with `the` too.